website/source/guides/quotas.html.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: "guides"
+page_title: "Resource Quotas"
+sidebar_current: "guides-quotas"
+description: |-
+  Nomad Enterprise provides support for resource quotas, which allows operators
+  to restrict the aggregate resource usage of namespaces.
+---
+
+# Resource Quotas
+
+Nomad Enterprise provides support for resource quotas, which allows operators to
+restrict the aggregate resource usage of namespaces.
+
+~> **Enterprise Only!** This functionality only exists in Nomad Enterprise.
+This is not present in the open source version of Nomad.
+
+## Use Case
+
+When many teams or users are sharing Nomad clusters, there is the concern that a
+single user could use more than their fair share of resources. Resource quotas
+provide a mechansim for cluster administrators to restrict the resources that a
+[namespace](/guides/namespace.html) has access to.
+
+## Quotas Objects
+
+Quota specifications are first class objects in Nomad. A quota specification
+has a unique name, an optional human readable description and a set of quota
+limits. The quota limits defines the allowed resource usage within a region.
+
+Quota objects are shareable among namespaces. This allows an operator to define
+higher level quota specifications, such as a `prod-api` quota, and multiple
+namespaces can apply the `prod-api` quota specification.
+
+When a quota specification is attached to a namespace, all resource usage by
+jobs in the namespaces are accounted toward the quota limits. If the resource is
+exhausted, allocations with the namespaces will be queued until resources become
+available by either other jobs finishing or the quota being expanded.
+
+## Working with Quotas
+
+For specific details about working with quotas, see the [quotas
```

Comment:
Maybe add something about how ACLs are useful to make sure quotas are not "self serve"? i.e, if ACLs are enabled, then only administrators with either a management token or a token with the correct permissions can create a quota.

Reply:
Good idea, added to the acl section.